### PR TITLE
added Linux Os Start to template and provided values

### PR DESCRIPTION
**Pull request by jaymarf**
Update of Linux Arm template to use Linux Os State property

### CODE REVIEW

File: ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json [hunk 1]
```diff
@@ -14,6 +14,18 @@
         "description": "Resource ID of an existing VM from which the custom image will be created."
       }
     },
+      "linuxOsState": {
```
Comment (drwill-ms) [inline ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json]: Can you open the file in VS Code and format the document (alt-shift-F)

File: ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json [hunk 2]
```diff
@@ -14,6 +14,18 @@
...
+      "type": "string",
+      "allowedValues": [
+        "NonDeprovisioned",
+        "DeprovisionRequested",
+        "DeprovisionApplied"
+        ],
+      "defaultValue": "NonDeprovisioned",
+      "metadata": {
+        "description": "State of Linux on the machine. It can be one of three values NonDeprovisioned, DeprovisionRequested, and DeprovisionApplied"
```
Comment (drwill-ms) [inline ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json]: end sentence with a period

File: ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json [hunk 3]
```diff
@@ -41,7 +53,10 @@
         "description": "[parameters('imageDescription')]",
         "osType": "Linux",
         "vm": {
-          "sourceVmId": "[parameters('existingVMResourceId')]"
+          "sourceVmId": "[parameters('existingVMResourceId')]",
+          "linuxOsInfo":{
+              "linuxOsState":  "[parameters('linuxOsState')]"
```
Comment (drwill-ms) [inline ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json]: minor: collapse the 2 spaces after the colon to 1

File: ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json (hunk 2 above)
Comment (drwill-ms) [inline ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json]: Not really just state. It's how we should treat it and prepare it for being a VM Image.

**Comment by mmydland** [comment]
This looks to be obsolete

**Pull request closed without merging**